crossdock/client/gauntlet/behavior.go

```diff
@@ -401,6 +416,11 @@ func RunGauntlet(t crossdock.T, c Config) {
 		t.Tag("service", tt.Service)
 		t.Tag("function", tt.Function)
 
+		//only run oneway tests if specified
+		if tt.Service == "ThriftTest" && !c.EnableOneway && tt.Oneway {
```

Comment:
you can drop the `== "ThriftTest"` condition. If a test wants to disable
oneway, it probably wants to disable all oneway.